packages/ia-zendesk-help-widget/src/ia-zendesk-help-widget.js

```diff
@@ -0,0 +1,75 @@
+import { html, LitElement } from 'lit-element';
+import zendeskWidgetCSS from './styles/zendesk-widget';
+import questionIcon from './icon-question-mark';
+
+class ZenDeskHelpWidget extends LitElement {
+  static get styles() {
+    return zendeskWidgetCSS;
+  }
+
+  static get properties() {
+    return {
+      widgetSrc: { type: String },
+      widgetkey: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.widgetSrc = '';
+    this.widgetkey = '';
+  }
+
+  initiateZenDesk() {
+    // Hide custom button
+    this.style.opacity = 0;
+
+    // prevent further if third party script already loaded
+    if (this.abortIfScriptAlreadyLoaded()) return;
+
+    // Inject help widget
+    this.injectScript();
```

Comment:
_Suggestion_ We have a javascript lazy loader service that allows you await the script loading like a promise so you can wait for it to load, then start your check for the button. ie:

Install it like this:
```
npm install @internetarchive/lazy-loader-service
```
then
```
import { LazyLoaderService } from '@internetarchive/lazy-loader-service';

async initiateZenDesk() {  <--- make this async
  const lazyLoaderService = new LazyLoaderService();
  await lazyLoaderService.loadScript({ src: this.widgetSrc });

  // the script is loaded, check for the button
}
```

It also won't double-load a script if it's already loaded so you won't have to check if the script has already been inserted.

Reply:
Thanks